sss/tests/test_sss.py

```diff
@@ -29,9 +29,14 @@ def test_check_extra_columns(setup_and_teardown_package):
 
     columns_to_check = ['miscellaneous_is_secondary', 'health_care_is_secondary', 'analysis_is_secondary']
 
-    for col in columns_to_check:
-        assert col in df.columns
-
+    is_df_empty = not df.empty
+    is_mics_empty = not miscellaneous.empty
+    is_hc_empty = not health_care.empty
+    is_arap_empty = not arpa.empty
+
+    assert is_df_empty, is_mics_empty
+    assert is_hc_empty, is_arap_empty
```

Comment:
This could be done a slightly more succinctly as:

```suggestion
    assert not df.empty
    assert not miscellaneous.empty
    assert not health_care.empty
   assert not arpa.empty
```

Reply:
These asserts are fine, but all they say is that there is **something** in the dataframes, not that the right things are in the dataframes.  A more rigorous check would be to assert that something you expect to be in the dataframes (in terms of exact quantities for a particular primary key set) was in the dataframe.